### PR TITLE
Support for custom Adhan and Fajr adhan

### PR DESCRIPTION
**Pull request by akhandok**
This PR proposes to add support for choosing a custom Adhan audio file to play. The PR also includes changes to play a different Adhan at Fajr that includes recitations specific for the Fajr Adhan.

The audio file I chose for Fajr is just one I picked out that I liked. Please let me know if you would like to change, or feel free to edit the PR.

Please note that testing was very rough. I am not able to install Docker because I have Windows 10 Home, so all my testing was done in place via Google Chrome's Developer Tools.

---

**Note**: this is a clone of [a previous PR](https://github.com/islamic-network/aladhan.com/pull/15) where the author of the commits were incorrect. This PR corrects the author and makes no other edits to the diff.

### CODE REVIEW

**Comment by meezaan** [comment]
@akhandok FYI I've just tried to run this but can't see the custom adhan file field at all. I'll take some more time to implement and test this out, but a a base, I think this is very useful. Thank you.

File: html/js/jquery.alAdhan.js [hunk 1]
```diff
@@ -243,9 +250,9 @@ jQuery( document ).ready( function( $ ) {
             }
             return theTime;
         },
-        doesPrayerTimeMatch: function(currentTime) {
+        getMatchingTiming: function(currentTime) {
```
Comment (meezaan) [inline html/js/jquery.alAdhan.js]: Can I ask the reason for this change? The method actually returns a boolean.

Comment (meezaan) [reply html/js/jquery.alAdhan.js]: Sorry, I see that's what has changed. It's now returning i.

**Comment by meezaan** [comment]
@akhandok I have refactored the code. It seems to work, but I need to leave it running for some time to make sure it works. I'll keep you posted.

**Comment by akhandok** [comment]
Ok, sounds good.

**Comment by meezaan** [comment]
Merged. Should be in production in a few minutes.

**Pull request merged**